renderer-macos/lib/Brisk.re

```diff
@@ -21,12 +20,12 @@ module NativeCocoa = {
 
   let insertNode = (~parent: node, ~child: node, ~position: int) => {
     Layout.cssNodeInsertChild(parent.layoutNode, child.layoutNode, position);
-    NSView.addSubview(parent.view, child.view);
+    BriskView.addSubview(parent.view, child.view);
```

Comment:
Nice. Although I believe addSubview stuff could be inside Cocoa as it's abstract for all views and cocoa even defines the view type.

Reply:
It could go either way, but it's for `BriskView`'s stub method, so might better be kept in appropriate binding file.

Reply:
I'm not sure! I don't think `BriskView` is the place for it because it's just a part of the common implementation. That said we can leave it like this for now!

Reply:
Ok, we can review this next time we revisit the bindings.